src/components/business/hireTalent.tsx/talentCard.tsx

```diff
@@ -0,0 +1,123 @@
+'use client';
+
+import React, { useState, useRef, useCallback } from 'react';
+import { Loader2 } from 'lucide-react';
+
+import { Avatar, AvatarFallback, AvatarImage } from '@/components/ui/avatar';
+import { Badge } from '@/components/ui/badge';
+import { Card, CardContent, CardHeader, CardTitle } from '@/components/ui/card';
+import { axiosInstance } from '@/lib/axiosinstance';
+import InfiniteScroll from '@/components/ui/infinite-scroll';
+
+interface DehixTalent {
+  _id: string;
+  skillName?: string;
+  domainName?: string;
+  experience: string;
+  monthlyPay: string;
+  status: string;
+  activeStatus: boolean;
+}
+
+interface Talent {
+  Name: string;
+  dehixTalent: DehixTalent;
+}
+
+const TalentCard: React.FC = () => {
+  const [talents, setTalents] = useState<Talent[]>([]);
+  const [skip, setSkip] = useState(0);
+  const [loading, setLoading] = useState(false);
+  const [hasMore, setHasMore] = useState(true);
+  const limit = 3; // Load 3 items per API request
+  const isRequestInProgress = useRef(false);
+
+  const fetchTalentData = useCallback(async () => {
+    if (isRequestInProgress.current || loading || !hasMore) return;
+
+    try {
+      isRequestInProgress.current = true;
+      setLoading(true);
+
+      const response = await axiosInstance.get(
+        `freelancer/dehixTalent?limit=${limit}&skip=${skip}`,
+      );
+
+      setTalents((prev) => [...prev, ...response.data.data]);
```

Comment:
check if you were able to get data successfully
add null checks just to be safe
use toast to display if there was error in fetching data